tools/workflow/is-commit-benchmarked.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
```

Comment:
Should we write this in go?

Reply:
I don't think it's necessary for all of our tooling to be in go. These tools are simple enough where it should be fine.